filesystem/06-windows-files.post

```diff
@@ -1,27 +1,26 @@
 maybe_create_winetc ()
 {
   local FILES="hosts protocols services networks"
-  local WINSYS32HOME="$(exec /usr/bin/cygpath -S -w)"
-  local WINETC="${WINSYS32HOME}\\drivers\\etc"
-
-  if [ ! -d "${WINETC}" ]; then
-    echo "Directory ${WINETC} does not exist; exiting" >&2
-    echo "If directory name is garbage you need to update your msys package" >&2
-    exit 1
-  fi
 
   local mketc=
   for mketc in ${FILES}
   do
     if [ ! -e "/etc/${mketc}" -a ! -L "/etc/${mketc}" ]
     then
+      local WINSYS32HOME="$(exec /usr/bin/cygpath -S -w)"
```

Comment:
This is now inside a loop.  I understand wanting to not call it unless it is needed but maybe the value could be cached so it only has to be done once?

Reply:
since it's only called on the first setup it didn't seem worth the extra logic.